From: Simon McVittie <smcv@collabora.com>
Date: Fri, 15 Jan 2021 17:41:33 +0000
Subject: waylandtouch: Don't export interface structs

These are explicitly written in C code rather than generated at build
time, so they weren't affected by changing how we invoke
wayland-scanner.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: https://bugzilla.libsdl.org/show_bug.cgi?id=5463
---
 src/video/wayland/SDL_waylandtouch.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/video/wayland/SDL_waylandtouch.c b/src/video/wayland/SDL_waylandtouch.c
index cfb95d2..cfddd08 100644
--- a/src/video/wayland/SDL_waylandtouch.c
+++ b/src/video/wayland/SDL_waylandtouch.c
@@ -160,7 +160,7 @@ static const struct wl_message qt_touch_extension_events[] = {
     { "configure", "u", qt_touch_extension_types + 0 },
 };
 
-WL_EXPORT const struct wl_interface qt_touch_extension_interface = {
+const struct wl_interface qt_touch_extension_interface = {
     "qt_touch_extension", 1,
     1, qt_touch_extension_requests,
     2, qt_touch_extension_events,
@@ -183,7 +183,7 @@ static const struct wl_message qt_windowmanager_events[] = {
     { "quit", "", qt_windowmanager_types + 0 },
 };
 
-WL_EXPORT const struct wl_interface qt_windowmanager_interface = {
+const struct wl_interface qt_windowmanager_interface = {
     "qt_windowmanager", 1,
     1, qt_windowmanager_requests,
     2, qt_windowmanager_events,
@@ -214,7 +214,7 @@ static const struct wl_message qt_surface_extension_requests[] = {
     { "get_extended_surface", "no", qt_surface_extension_types + 2 },
 };
 
-WL_EXPORT const struct wl_interface qt_surface_extension_interface = {
+const struct wl_interface qt_surface_extension_interface = {
     "qt_surface_extension", 1,
     1, qt_surface_extension_requests,
     0, NULL,
@@ -232,7 +232,7 @@ static const struct wl_message qt_extended_surface_events[] = {
     { "close", "", qt_surface_extension_types + 0 },
 };
 
-WL_EXPORT const struct wl_interface qt_extended_surface_interface = {
+const struct wl_interface qt_extended_surface_interface = {
     "qt_extended_surface", 1,
     3, qt_extended_surface_requests,
     3, qt_extended_surface_events,
